test/studies/shootout/submitted/nbody4.perfkeys

```diff
@@ -0,0 +1 @@
+nbody3.perfkeys
```

Comment:
I'm guessing you meant to copy past `nbody3.perfkeys` here?

Reply:
Huh… these were symbolic links, which I thought were OK to commit to git…  Is this just how GitHub renders them?  If you grab a copy of the branch, do you get the links out?

Reply:
Ah, I think the rendered contents of a symbolic link _is_ the path that the link is to?  So presumably it's the meta-information that makes this file "work right" when checked out.

Reply:
Oh I see, in that case never mind, everything looks good!